File: geeksforgeeks/w3-p5-nth-node-from-end-of-linked-list/Solution.c
int getKthFromLast(struct Node* head, int k) {
    struct Node* curr = head;
    int count = 0;
    while(curr) {
        curr = curr->next;
        count++;
    }
    if(k>count) {
        return -1;
    } else {
        for(int i=0; i<(count-k); i++) {
            head = head->next;
        }
        return head->data;
    }
}

File: geeksforgeeks/w3-p5-nth-node-from-end-of-linked-list/Solution.py
'''
	{
		# Node Class
		class Node:
		    def __init__(self, data):   # data -> value stored in node
		        self.data = data
		        self.next = None
	}
'''
class Solution:
    def getKthFromLast(self, head, k):
        curr = head
        count = 0
        while curr:
            curr = curr.next
            count = count+1
        if k>count:
            return -1
        else:
            for x in range(count-k):
                head = head.next
            return head.data